cmd/kubeadm/app/apis/kubeadm/v1alpha1/zz_generated.conversion.go

```diff
@@ -16,6 +16,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Code generated by conversion-gen. DO NOT EDIT.
+
 // This file was autogenerated by conversion-gen. Do not edit it manually!
```

Comment:
If we add the message to the framework, we need to remove it from the individual generators? Otherwise it's very stuttery...

Reply:
Thanks for pointing that out.
I'll have to remember that when we update the vendored gengo.